src/lib/core/CHIPConfig.h

```diff
@@ -1460,6 +1460,42 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 #define CHIP_CONFIG_SKIP_APP_SPECIFIC_GENERATED_HEADER_INCLUDES 0
 #endif
 
+/**
+ * @def CHIP_CONFIG_ICD_IDLE_MODE_INTERVAL
+ *
+ * @brief Default value for the ICD Management cluster IdleModeInterval attribute, in milliseconds
+ */
+#ifndef CHIP_CONFIG_ICD_IDLE_MODE_INTERVAL
```

Comment:
please add `_MS` and `_SEC` and similar units. Makes it clear whenever used without needing to try to follow the comment.

Reply:
@andy31415 Opened PR https://github.com/project-chip/connectedhomeip/pull/29417